boards/arm/nrf9160_pca10090/nrf9160_pca10090ns.dts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf9160ns_sica.dtsi>
+#include "nrf9160_pca10090_common.dts"
+
+/ {
+	chosen {
+		zephyr,flash = &flash0;
+		zephyr,sram = &sram0_ns;
+		zephyr,code-partition = &slot0_ns_partition;
```

Comment:
How does this play with `dts/common/mcuboot.overlay` that specifies
`zephyr,code-partition = &slot0_partition;`
What is the precedence?

Reply:
That is correct.

Reply:
@ioannisg this thread has not been resolved 🐛

Reply:
MCUboot should not be allowed for Non-Secure images - this is something we could agree on, right? [Sorry, mistake to resolve that one]

Reply:
We cleared this up. For now this is fine as it is, but non-secure firmware images that wish to be programmed using MCUboot need to leave the "MCUBoot support" option unchecked, otherwise MCUBoot overlay will override the code partition and they will be linked in the secure one.